operators-manual/companion-web.md

```diff
@@ -10,19 +16,26 @@ The web interface by default opens at the [Network](http://192.168.2.2:2770/netw
 - Check WiFi and Internet Status 
 - Configure Ethernet IP address
 - Test the network for Uploads and Downloads
+- Ethernet network configuration (advanced): The Ethernet interface can be configured as DHCP Server, DHCP Client or Manual (static) mode.
+  - Manual: Manual: The interface is configured with a static IP address. This is the default configuration. The default IP address is 192.168.2.2.
+  - DHCP Server: The interface is configured as a DHCP server on the 192.168.2.x subnet. The interface IP address will be 192.168.2.2 in this mode.
+  - DHCP Client: This is useful for connecting to a router.
 
-<img src="/images/network-setup.png" class="img-responsive img-center" style="max-width:750px" />
+<img src="/images/Network.png" class="img-responsive img-center" style="max-width:750px" />
 
 ##System
 
 The [System](http://192.168.2.2:2770/system) page provides the following functionalities and features:
 
-- Display companion computer hardware status parameters
-- Display software version
-- Update software on companion
+- Display Companion computer hardware status parameters and detailed status of processes.
```

Comment:
Did we lose update companion software?

Reply:
yeah.. It was your suggested edit so i removed it. I also thought that it should be there but then it was your suggested edit so I thought you wanted to remove it. I will add it again because it should be there.